File: sqlInsert.py
import MySQLdb

#create connection
db = MySQLdb.connect(host="localhost", user="user", passwd="password", db="school")

#create cursor
cur = db.cursor(MySQLdb.cursors.DictCursor)
db.autocommit(True)

sql = "INSERT INTO students (name,age,gradeLevel) VALUES ('Kaiki','16','11')"

cur.execute(sql)

cur.close()
db.close()

File: sqlInsert.php
<?php
$servername = "localhost";
$username = "User";
$password = "password";
$dbname = "school";
$name = 'Aaron';
$age = 17;
$gradeLevel = 11;
$conn = new mysqli($servername, $username, $password, $dbname);

// Check connection
if ($conn->connect_error) {
    die("Connection failed: " . $conn->connect_error);
}
echo "Connected successfully";

$sql = "INSERT INTO students (name, age, gradeLevel)
VALUES ('$name', '$age', '$gradeLevel')";

if ($conn->query($sql) === TRUE) {
    echo "New record created successfully";
} else {
    echo "Error: " . $sql . "<br>" . $conn->error;
}

$conn->close();
?>
